spec/rails_helper.rb

```diff
@@ -5,12 +5,33 @@
 require 'capybara/rspec'
 require 'capybara/rails'
 require 'webmock/rspec'
+require 'database_cleaner'
 
 Capybara.javascript_driver = :webkit
 
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.start
+  end
+
+  config.append_after(:each) do
+    DatabaseCleaner.clean
+  end
+
```

Comment:
Why do you need database cleaner. I'd be very careful with that and only use it if you really need it.